app/src/main/java/com/example/justjet/Ui/HomeFragment.java

```diff
@@ -0,0 +1,116 @@
+package com.example.justjet.Ui;
+
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProviders;
+import androidx.navigation.NavController;
+import androidx.navigation.Navigation;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.ListView;
+
+import com.example.justjet.R;
+import com.example.justjet.Models.ViewModel;
+import com.google.firebase.database.DataSnapshot;
+
+import java.util.ArrayList;
+
+public class HomeFragment extends Fragment {
+
+    private ListView Listt;
```

Comment:
@Riyabelle25 You have to use Recycler view. Ask in this thread if you have any queries regarding recycler view.